Website/index.js

```diff
@@ -0,0 +1,474 @@
+            $("#second_on_break").text(listToString(data["on_break"]));
+        }
+
+        function thirdRecess() {
+            var data = jsonData["presets"][jsonData["dates"][String($("#start").val())]]["recess_duty"]["third_recess"]
+            $("#third_nb1").text(listToString(data["nb1"]));
+            $("#third_nb2").text(listToString(data["nb2"]));
+            $("#third_nb3").text(listToString(data["nb3"]));
+            $("#third_stall").text(listToString(data["stall"]));
+            $("#third_pond").text(listToString(data["pond"]));
+            $("#third_lift").text(listToString(data["lift"]));
+            $("#third_central").text(listToString(data["central"]));
+            $("#third_on_break").text(listToString(data["on_break"]));
+        }
+
+        function fourthRecess() {
+            var data = jsonData["presets"][jsonData["dates"][String($("#start").val())]]["recess_duty"]["fourth_recess"];
+            $("#fourth_nb1").text(listToString(data["nb1"]));
+            $("#fourth_nb2").text(listToString(data["nb2"]));
+            $("#fourth_nb3").text(listToString(data["nb3"]));
+            $("#fourth_stall").text(listToString(data["stall"]));
+            $("#fourth_pond").text(listToString(data["pond"]));
+            $("#fourth_lift").text(listToString(data["lift"]));
+            $("#fourth_central").text(listToString(data["central"]));
+            $("#fourth_on_break").text(listToString(data["on_break"]));
+        }
+
+        firstRecess();
+        secondRecess();
+        thirdRecess();
+        fourthRecess();
+    }
+
+    function checkTimeValid(start, end) {
+        var startTime = String(start);
+        var endTime = String(end);
+
+        currentDate = new Date();
+
+        startDate = new Date(currentDate.getTime());
+        startDate.setHours(startTime.split(":")[0]);
+        startDate.setMinutes(startTime.split(":")[1]);
+        startDate.setSeconds(startTime.split(":")[2]);
+
+        endDate = new Date(currentDate.getTime());
+        endDate.setHours(endTime.split(":")[0]);
+        endDate.setMinutes(endTime.split(":")[1]);
+        endDate.setSeconds(endTime.split(":")[2]);
+
+
+        var valid = startDate < currentDate && endDate > currentDate
+        return valid;
+    }
+
+    if (checkTimeValid("09:10:00", "11:30:00") && today(String($("#start").val()))) {
+        if (checkTimeValid("09:10:00", "09:45:00") && check1 === false) {
+            recessTime(1)
+            check1 = true;
+        } else if (checkTimeValid("09:45:00", "10:20:00") && check1 === false) {
+            recessTime(2);
+            check1 = true;
+        } else if (checkTimeValid("10:20:00", "10:55:00") && check1 === false) {
+            recessTime(3)
+            check1 = true;
+        } else if (checkTimeValid("10:55:00", "11:30:00") && check1 === false) {
+            recessTime(4)
+            check1 = true;
+        }
+    } else if (blink === true) {
+        $('.icon-blink').each(function () {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
        $(".icon-blink").each(function () {
```